package.json

```diff
@@ -15,6 +15,7 @@
     "clean:root": "shx rm -rf node_modules",
     "update:commit-scopes": "node scripts/generateCommitScopeEnum.js",
     "lint": "lerna run lint",
```

Comment:
yarn lint still exits with errors. For a better overview you can execute in in the respective packages